packages/@okta/i18n/src/properties/login.properties

```diff
@@ -509,30 +512,37 @@ enroll.webauthn.instructions.noSupportForBiometric = Note: Some browsers may not
 authfactor.webauthn.error = You are currently unable to use a Security key or biometric authenticator. Try again.
 
 ## Enter Passcode Form
-enroll.totp.enterCode = Enter code displayed from the application
+enroll.totp.enterCode = Enter temporary code generated from the application
 
 ## Barcode View
 # {0} is the name of a MFA factor
 enroll.totp.setupApp = Launch {0} on your mobile device and select “Add an account”. Scan the QR code to continue.
 # {0} is the name of a MFA factor
 enroll.totp.setupGoogleAuthApp = Launch {0}, tap the "+" icon, then select "Scan barcode".
 enroll.totp.cannotScan = Can't scan?
-enroll.totp.aria.cannotScan = Setup without scanning a QR code.
+enroll.totp.setupWithoutQRCode = Setup without scanning a QR code.
 enroll.totp.refreshBarcode = Refresh code
 
 ## Manually Setup Push
-enroll.totp.cannotScanBarcode = Can't scan barcode?
+enroll.totp.moreOptions = More options
```

Comment:
same here, let's change the existing one `enroll.totp.cannotScanBarcode`

Reply:
done